receiver/pulsarreceiver/README.md

```diff
@@ -0,0 +1,65 @@
+# Pulsar Receiver
+
+| Status                   |                       |
+| ------------------------ |-----------------------|
+| Stability                | [in-development]      |
```

Comment:
It should be `alpha` status if your PR provides a functional and you enable the component